src/ucp/api/ucp.h

```diff
@@ -255,6 +256,36 @@ enum {
 
 
 /**
+ * @ingroup UCP_COMM
+ * @brief Atomic operation requested for ucp_atomic_post
+ *
+ * This enumeration defines which atomic memory operation should be
+ * performed by the ucp_atomic_post family of fuctions. All of these are
+ * non-fetching atomics and will not result in a request handle.
+ */
+typedef enum {
+    UCP_ATOMIC_POST_OP_ADD, /**< Atomic add */
+    UCP_ATOMIC_POST_OP_LAST
+} ucp_atomic_post_op_t;
+
+
+/**
+ * @ingroup UCP_COMM
+ * @brief Atomic operation requested for ucp_atomic_fetch
+ *
+ * This enumeration defines which atomic memory operation should be performed
+ * by the ucp_atomic_fetch family of functions. All of these functions
+ * will fetch data from the remote node.
+ */
+typedef enum {
+    UCP_ATOMIC_FETCH_OP_FADD, /**< Atomic Fetch and add */
+    UCP_ATOMIC_FETCH_OP_SWAP, /**< Atomic swap */
+    UCP_ATOMIC_FETCH_OP_CSWAP, /**< Atomic conditional swap */
+    UCP_ATOMIC_FETCH_OP_LAST
+} ucp_atomic_fetch_op_t;
```

Comment:
i found that having this enum in between function declarations is what creates the page break in ucx.pdf that @bbenton noticed